src/amadeus/namespaces/booking/flight_orders.js

```diff
@@ -28,7 +28,7 @@ class FlightOrders {
    * amadeus.booking.flightOrders.post({
    *  'type': 'flight-order',
    *  'flightOffers': [],
-   *  'travelers_info': []
+   *  'traveler': []
```

Comment:
It is `travelers` according to the [specification](https://developers.amadeus.com/self-service/category/flights/api-doc/flight-create-orders/api-reference)